### PR TITLE
fix(i18n): i18n fixes

### PR DESCRIPTION
**Pull request by BlackHole1**

### CODE REVIEW

File: packages/flat-i18n/locales/en.json [hunk 1]
```diff
@@ -364,7 +365,8 @@
     "choose-share-content": "Choose what to share",
     "tip-window-title": "Ending screen sharing",
     "tip-window-body": "You are sharing the current screen",
-    "tip-window-button": "End Sharing"
+    "tip-window-button": "End Sharing",
```
Comment (Cheerego7) [inline packages/flat-i18n/locales/en.json:367]: `End` -> `Stop`

**Comment by BlackHole1** [comment]
@Cheerego7 please re-review

File: packages/flat-i18n/locales/en.json [hunk 2]
```diff
@@ -133,7 +133,8 @@
   "room-type": "Room Type",
   "think-again": "Think again",
   "cancel-general-room-tips": "Are you sure to cancel this room? \nAfter cancellation, other members will not be able to join.",
-  "copy-invitation": "Invite",
+  "invitation": "Invite",
+  "copy-invitation": "Copy Invite",
```
Comment (LitoMore) [inline packages/flat-i18n/locales/en.json]: `Copy Invite` is the wrong syntax.

```suggestion
  "copy-invitation": "Copy Invite Link",
```

Comment (LitoMore) [reply packages/flat-i18n/locales/en.json]: `Copy Invite Link` and `Copy Invitation` are different. Here is the Invite UI of Zoom:

image

And here is the difference between `Copy Invite Link` and `Copy Invitation`:

#### Copy Invite Link

```
https://zoom.example.com/j/123456789?pwd=abcdef
```

#### Copy Invitation

```
Join Zoom Meeting
https://zoom.example.com/j/123456789?pwd=abcdef

Meeting ID: 123 456 789
Passcode: abcdef
One tap mobile
+123123123,,123123123#,,,,*123# US (New York)
+123123123,,123123123#,,,,*123# US (San Jose)

Dial by your location
        +1 123 123 1234 US (New York)
        +1 123 123 1234 US (San Jose)
        +1 123 123 1234 US (Tacoma)
        +1 123 123 1234 US (Washington DC)
        +1 123 123 1234 US (Chicago)
        +1 123 123 1234 US (Houston)
Meeting ID: 123 123 123
Passcode: abcdef
Find your local number: https://zoom.example.com/abcdef

Join by SIP
123123123@example.com

Join by A.123
123.123.123.123 (US West)
123.123.123.123 (US East)
123.123.123.123 (China)
123.123.123.123 (India Mumbai)
123.123.123.123 (India Hyderabad)
123.123.123.123 (Amsterdam Netherlands)
123.123.123.123 (Germany)
123.123.123.123 (Australia Sydney)
123.123.123.123 (Australia Melbourne)
123.123.123.123 (Hong Kong SAR)
123.123.123.123 (Singapore)
123.123.123.123 (Brazil)
123.123.123.123 (Canada Toronto)
123.123.123.123 (Canada Vancouver)
123.123.123.123 (Japan Tokyo)
123.123.123.123 (Japan Osaka)
Meeting ID: 123 456 789
Passcode: abcdef
```

Comment (BlackHole1) [reply packages/flat-i18n/locales/en.json]: thank you for your careful review 🙇.  I find that our understanding is all incorrect.

Here it is not a "click and copy" function, but "click and a pop-up window appears with a copy". 

e.g:

image

image

I've just spoken to @Leooeloel  and our solution is to remove the `copy-invitation` and just keep the `invitation`

Comment (LitoMore) [reply packages/flat-i18n/locales/en.json]: > Here it is not a "click and copy" function, but "click and a pop-up window appears with a copy".

Then its text should be `Invite`.

Comment (BlackHole1) [reply packages/flat-i18n/locales/en.json]: Yes. I will soon be submitting a PR

**Review by LitoMore — APPROVED** [review]

**Pull request merged**